tests/dtm/test_single_process_restart.py

```diff
@@ -66,6 +69,10 @@ def setup_class(cls):
         cls.test_cfg = configmanager.get_config_wrapper(fpath="config/test_dtm_config.yaml")
         cls.m0d_process = 'm0d'
         cls.dtm_obj = DTMRecoveryTestLib()
+        cls.s3_obj = S3TestLib()
+        cls.dir_path = os.path.join(os.getcwd(), TEST_DATA_FOLDER, "dtm")
+        if not os.path.exists(cls.dir_path):
+            os.makedirs(cls.dir_path)
```

Comment:
Directory for dtm test data is added in PR#1582. You will get it once that PR is merged.

Reply:
will change once mentioned PR gets merged.